src/main/scala/org/apache/spark/sql/hive/llap/DefaultSource.scala

```diff
@@ -17,16 +17,23 @@
 
 package org.apache.spark.sql.hive.llap
```

Comment:
Ya. I thought like that at first sight. But, it cannot be there because it needs to access `sharedState` to get `catalog`.

Actually, I moved all classes into `com.hortonworks` at the first time. After that, I minimally moved back into `org.apache` in order to access `package private` members.